lib/plugins/create/templates/azure-nodejs/serverless.yml

```diff
@@ -0,0 +1,53 @@
+# Welcome to Serverless!
+#
+# This file is the main config file for your service.
+# It's very minimal at this point and uses default values.
+# You can always add more config options for more control.
+# We've included some commented out config examples here.
+# Just uncomment any of them to get that config option.
+#
+# For full config options, check the docs:
+#    docs.serverless.com
+#
+# Happy Coding!
+
+service: azure-nodejs # NOTE: update this with your service name
+
+# You can pin your service to only deploy with a specific Serverless version
+# Check out our docs for more details
+# frameworkVersion: "=X.X.X"
+
+provider:
+  name: azure
+  location: West US
+
+plugins:
+  - serverless-azure-functions
+
+# you can add packaging information here
+#package:
+#  include:
+#    - include-me.js
+#    - include-me-dir/**
+#  exclude:
+#    - exclude-me.js
+#    - exclude-me-dir/**
+
+functions:
+  hello: 
+    handler: handler.hello
+    events: 
+      - http: true
```

Comment:
The `handler.js` code is fairly coupled to the event type though (even for AWS?), so it seemed to make sense to have the `serverless.yml` actually be functional from the start. Maybe `aws` defaults to `http` if an event isn't otherwise specified?

The AWS file you mentioned says that you'd need to change `handler.js` to adjust to any event changes. From that perspective, I was thinking it made sense to have the event definition be functional, and then have a comment that illustrates some other event types you can try. That makes the getting started/evaluation process super simple, and then you can tweak it based on your needs.